iroh-metrics/src/config.rs

```diff
@@ -27,8 +27,10 @@ impl Config {
         service_env: String,
         debug: bool,
     ) -> Self {
-        let debug =
-            std::env::var("IROH_METRICS_DEBUG").unwrap_or_else(|_| debug.to_string()) == "true";
+        let debug: bool = std::env::var("IROH_METRICS_DEBUG")
+            .unwrap_or_else(|_| debug.to_string())
```

Comment:
no need to convert debug to a string

Reply:
How, it's a compile type error if I don't as the `env::var` returns a `Result<String, error>`. Have to match return types on unwrap.

Reply:
```rust
let debug: bool = std::env::var("IROH_METRICS_DEBUG")
  .and_then(|v| v.parse().ok())
  .unwrap_or_default();
```

Reply:
Also doesn't work :/ 
`std::env::var("IROH_METRICS_DEBUG")` produces `Result<String, VarError>` which means that `.and_then` needs to produce the same type so that the final `unwrap_or` can consume either as they are the same type.

Reply:
ah sorry, you need to add an .ok() before the and_then, to turn it into an option

Reply:
We'll this has been a fun rust exercise in parsing :D ty